x-pack/plugins/siem/cypress/integration/signal_detection_rules_custom.spec.ts

```diff
@@ -80,7 +80,7 @@ describe('Signal detection rules, custom', () => {
     goToManageSignalDetectionRules();
     waitForLoadElasticPrebuiltDetectionRulesTableToBeLoaded();
     goToCreateNewRule();
-    fillDefineCustomRuleAndContinue(newRule);
+    fillDefineCustomRuleWithImportedQueryAndContinue(newRule);
```

Comment:
Do we have another test that exercises manual entry of a query? I don't think we should stop testing that functionality.

Reply:
Is it possible to have that covered with a jest test? if not we can have 2 different tests for rule creation :)

Reply:
I think a jest test is a good compromise. It'll have to simulate an onChange event instead of real user input so there's potential for false positive, but between the two I think that'll be good 👍

Reply:
but `fillDefineCustomRuleAndContinue` is now dead code, right?

Reply:
You are right!! But I'm maintaining that function since I know that the creation rules functionality is going to be extended and I'm not sure if it could be helpful. If I see that is not useful, I'll delete it.